benches/oneshot/ipa.rs

```diff
@@ -111,16 +111,36 @@ async fn run(args: Args) -> Result<(), Error> {
         q = args.query_size
     );
     let rng = StdRng::seed_from_u64(seed);
+    let (user_count, min_events_per_user, max_events_per_user, query_size) =
+        if args.oprf && cfg!(feature = "step-trace") {
+            // For the steps collection, OPRF mode requires a single user with the same number
+            // of dynamic steps as defined for `UserNthRowStep::Row`.
+            (
+                NonZeroU64::new(1).unwrap(),
+                NonZeroU32::new(64).unwrap(),
+                NonZeroU32::new(64).unwrap(),
+                64,
```

Comment:
One cannot tell by looking at this how these numbers are used, and what they mean.

Reply:
Could you tell me which part is unclear? I think the tuple variable names are self-explanatory and the comment describes why we need 1 and 64. Maybe change the integer literals to constants?